scraper/src/mindtouch2zim/processor.py

```diff
@@ -427,6 +427,8 @@ def _process_css(
         """Process a given CSS stylesheet
         Download content if necessary, rewrite CSS and add CSS to ZIM
         """
+        CONTEXT.processing_step = f"CSS at {css_location}"
+        logger.debug(f"  Fetching {CONTEXT.processing_step}")
```

Comment:
Shouldn't changing the value of processing_step automatically trigger a debug log? Would be more solid. Can be done with a setter